src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithplaintextcommentfilter/InputSuppressWithPlainTextCommentFilterWithCustomOnComment.java

```diff
@@ -0,0 +1,6 @@
+package com.puppycrawl.tools.checkstyle.filters.suppresswithplaintextcommentfilter;
+
+public class InputSuppressWithPlainTextCommentFilterWithCustomOnComment {
+    // cs-on
+    //	has tab here
```

Comment:
Does it really need a tab character? We have an issue for this: https://github.com/checkstyle/checkstyle/issues/5941

Reply:
I added a SQL file instead with the example "COUNT(*)" check from the docs